src/CW/Block/Block.php

```diff
@@ -60,4 +60,22 @@ public static function getHookBlockViewArray() {
    */
   abstract function getRenderArray();
 
+  /**
```

Comment:
This seems like a utility function. This class is to define block instances. If it's not related to a block instance strictly, better to move to a generic level class: CW\Util\BlockUtil, as an example.